crypto/types/multisig/threshold_pubkey_test.go

```diff
@@ -180,7 +235,12 @@ func generatePubKeysAndSignatures(n int, msg []byte) (pubkeys []crypto.PubKey, s
 			privkey = sr25519.GenPrivKey()
 		}
 		pubkeys[i] = privkey.PubKey()
-		signatures[i], _ = privkey.Sign(msg)
+		sig, _ := privkey.Sign(msg)
+		var multisignature signing.SignatureData
+		cdc.UnmarshalBinaryBare(sig, &multisignature)
```

Comment:
We can't do marshaling `SignatureData`. It's a application logic, not encoding type.